recipes/extern/meta.yaml

```
@@ -0,0 +1,37 @@
+{% set name = "extern" %}
+{% set version = "0.4.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "0ff01adc2ad423f3d1e31641024b3974569fb0127b4d925bc6bed1cb86b6b1e4"
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - pip
+    - python >=3
+  run:
+    - python >=3
+
+test:
+  imports:
+    - extern
+
+about:
+  home: "The package home page"
```

Looks like there isn't much of an home page or dev/doc url available. Let's just go with
```suggestion
  home: https://pypi.org/project/extern/
```
then.